counter.code-workspace

```diff
@@ -0,0 +1,8 @@
+{
+	"folders": [
+		{
+			"path": "."
+		}
+	],
+	"settings": {}
+}
```

Comment:
was ist das 🤔

Reply:
That would be my workspace in VS Code, so if I have to close the window for some reason I can just open that up and all the files I'm working on will be there.